### PR TITLE
fix(@aws-amplify/interactions): remove `Readable` dependencies

### PR DESCRIPTION
**Pull request by wlee221**
_Issue #, if available:_

_Description of changes:_

This removes `Readable` dependency from the AWS Lex provider and unit testing.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: packages/interactions/src/Providers/AWSLexProviderHelper/convert.ts [hunk 1]
```diff
@@ -1,41 +1,9 @@
-import { Readable } from 'stream';
-export const convert = async (
-	stream: Readable | ReadableStream | Blob
-): Promise<Uint8Array> => {
-	if (stream instanceof Readable) {
-		return readOnNode(stream);
+export const convert = (stream: object): Promise<Uint8Array> => {
```
Comment (ashika01) [inline packages/interactions/src/Providers/AWSLexProviderHelper/convert.ts:1]: Could we still keep the `stream` type to be `Readable | ReadableStream | Blob` ??

**Comment by ashika01** [comment]
Just nit! Looking good overall?!

File: packages/interactions/src/Providers/AWSLexProviderHelper/convert.ts (hunk 1 above)
Comment (wlee221) [reply packages/interactions/src/Providers/AWSLexProviderHelper/convert.ts:1]: That should be fine as long as I don't do `instanceof Readable`; I'll add that in.

**Comment by wlee221** [comment]
Thanks @jordanranz @ashika01 @elorzafe 👍

**Pull request merged**